test-framework/test-suites/system/tests/stacki/test_frontend.py

```diff
@@ -1,6 +1,8 @@
-import pytest
+from collections import defaultdict
 import json
 
+import pytest
```

Comment:
This pytest import appears unnecessary?

Reply:
If it is I'll remove it. I was just cleaning up the import order at the top of the file.